### PR TITLE
RDK-28320: Enable Thunder Packager plugin

### PR DESCRIPTION
**Pull request by tdeva14**
Reason for change:
1. Makefile changes to build Packager plugin
2. libprovision dependency not required
3. Add cmake config file for OPKG
Test Procedure: Build Packager plugin
Risks: None

Signed-off-by: Deva <Deva_Thiyagarajan2@comcast.com>

### CODE REVIEW

**Comment by vijs** [comment]
@tdeva14 Can you confirm if desktop build is successful?

File: Packager/CMakeLists.txt [hunk 1]
```diff
@@ -18,8 +18,9 @@
 set(PLUGIN_NAME Packager)
 set(MODULE_NAME WPEFramework${PLUGIN_NAME})
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
 find_package(${NAMESPACE}Plugins REQUIRED)
-find_package(libprovision REQUIRED)
```
Comment (pwielders) [inline Packager/CMakeLists.txt:22]: Can we please make this conditional. We have customers using this packager relying on libprovision in combination with OPKG. So suggest to make this package optional, and if found, build it the old way, if not do it the comcast used way.
This way you do not break current deplyments

**Comment by tdeva14** [comment]
@pwielders  @vijs Made the changed conditional. Could you please review and merge ?

File: Packager/CMakeLists.txt [hunk 2]
```diff
@@ -18,7 +18,11 @@
 set(PLUGIN_NAME Packager)
 set(MODULE_NAME WPEFramework${PLUGIN_NAME})
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+if (BUILD_RDK)
```
Comment (pwielders) [inline Packager/CMakeLists.txt:21]: Why do we want to make this explicit ? Why can we not just say:
find_package(libprovision QUIET)

if (libprovision_FOUND)
  ...
else ()
   ...
endif ()

This way it work automagically if you install libprovision

**Pull request merged**